hack/logfile_metric_daemonset.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
```

Comment:
This deployment belongs to `config/overlays/openshift/storage-size-calculator` like the rest of the manifests.